cmd/fleetctl/get.go

```diff
@@ -899,3 +900,75 @@ func getTeamsCommand() *cli.Command {
 		},
 	}
 }
+
+func getSoftwareCommand() *cli.Command {
+	return &cli.Command{
+		Name:    "software",
+		Aliases: []string{"s"},
+		Usage:   "List software",
+		Flags: []cli.Flag{
+			&cli.UintFlag{
+				Name:  teamFlagName,
+				Usage: "Only list software of hosts that belong to the specified team",
+			},
+			jsonFlag(),
+			yamlFlag(),
+			configFlag(),
+			contextFlag(),
+			debugFlag(),
+		},
+		Action: func(c *cli.Context) error {
+			client, err := clientFromCLI(c)
+			if err != nil {
+				return err
+			}
+
+			var teamID *uint
+
+			teamIDFlag := c.Uint(teamFlagName)
+			if teamIDFlag != 0 {
+				teamID = &teamIDFlag
+			}
+
+			software, err := client.ListSoftware(teamID)
+			if err != nil {
+				return errors.Wrap(err, "could not list software")
+			}
+
+			if len(software) == 0 {
+				log(c, "No software found")
+				return nil
+			}
+
+			if c.Bool(jsonFlagName) || c.Bool(yamlFlagName) {
```

Comment:
It should probably be an error if both are specified, though that may be handled elsewhere?